src/commands/promote.js

```diff
@@ -2,6 +2,19 @@ const colors = require('colors/safe');
 const utils = require('../utils');
 const _ = require('lodash');
 
+const runAppReviewChecks = async (appId, appVersion, context) => {
+  context.line('\nRunning App Review Checks.\n');
+  const url = `/apps/${appId}/versions/${appVersion}/app-review-run`;
+  const response = await utils.callAPI(url, true);
+  const errorList = response.failed || [];
+  if (errorList.length > 0) {
+    const message = `Promotion failed for the following reasons:\n\n${errorList
```

Comment:
I was going back and forth between whether this should say `Activation failed` or `Promotion failed`, since if they see it, they were trying to activate. Saying `promotion` make it sound like there was an error with promotion rather than it being blocked. 

Your call, not sure we even need a change here.